packages/buildbucket-dart/CHANGELOG.md

```diff
@@ -0,0 +1,3 @@
+# 0.0.1
```

Comment:
Can we make this a 1.0.0?

Pre <1.0.0 packages are a pain to upgrade, as every version increment is considering a breaking change.

Reply:
Done